private/bufpkg/bufmodule/bufmoduleapi/uploader.go

```diff
@@ -443,6 +429,20 @@ func getV1Beta1ProtoUploadRequestContent(
 	}, nil
 }
 
+func validateModuleDefaultLabels(modules []*modulev1.Module) error {
```

Comment:
Repurpose to be `getSortedModuleDefaultLabels(...) []string` and use `len > 1` above